app/controllers/application_controller.rb

```diff
@@ -13,4 +13,9 @@ class ApplicationController < ActionController::Base
   ActionController::Parameters.permit_all_parameters = true
 
   helper_method :blackcat_config
+
+  before_action do
+    # @todo Need to limit this to admin users only
+    Rack::MiniProfiler.authorize_request
```

Comment:
@whereismyjetpack How would we limit this to our "admin" users? Do we have access to group information for the current user?

Reply:
For blacklight, we don't. We'd have to hook into devise to make decisions at this layer. We can protect a path at the ingress layer, i.e  `/profilier` let me do some more research

Reply:
We don't necessarily have to use Devise. We could just do it via session information. When a new session is started, it checks the request for the omniauth information. Is that available currently?

Reply:
I'm opting for an ENV variable to enable this only in dev and preview instances until we can implement user-based restrictions.